server/src/server.js

```diff
@@ -19,22 +19,24 @@ app.get('/', (req, res) => {
 /**
   Simple flight search api wrapper.
 
-  TODO: client should provide params
+  TODO: client should provide params.
```

Comment:
I like the level of commitment to improve user experience demonstrated here 😂

Reply:
I'm a bit of a stickler for correct grammar in user-facing content 😂